backend/estimate-service/src/main/java/org/egov/works/util/MDMSUtils.java

```diff
@@ -77,6 +78,11 @@ public MdmsCriteriaReq getMDMSRequest(RequestInfo requestInfo, String tenantId,
         //moduleDetails.add(estimateLocationModuleDetail);
         moduleDetails.add(estimateTenantModuleDetail);
 
+        //split the tenantId
+        if(StringUtils.isNotBlank(tenantId)){
+            tenantId = tenantId.split("\\.")[0];
```

Comment:
Don't write this at this stage. Make different function calls for different tenants. Don't split tenant within mdms. Split it begore making this function call.

Reply:
Yes, I have moved to the caller method.

Reply:
Then remove these 3 lines of code from here. MDMSUtil should not split the tenant.